src/core_plugins/kibana/public/context/query_parameters/actions.js

```diff
@@ -0,0 +1,57 @@
+import _ from 'lodash';
+
+import {
+  MAX_CONTEXT_SIZE,
+  MIN_CONTEXT_SIZE,
+  QUERY_PARAMETER_KEYS,
+} from './constants';
+
+
+export function QueryParameterActionsProvider() {
```

Comment:
Not really, but I left it as a provider to maintain consistency with the query actions. The pattern was "actions are created via a provider (to inject required angular dependencies)". It could become "actions are created via a provider if they have angular dependencies" instead. I'm not partial to either.